src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/PopulateStochasticBurdenEstimateSetTests.kt

```diff
@@ -0,0 +1,263 @@
+package org.vaccineimpact.api.databaseTests.tests
```

Comment:
Have gone with duplicating all the existing tests, just using a stochastic set instead. The stochastic estimate writer & central estimate writer are tested implicitly through these 2 test classes. I think I'm ok with this, although it would probably make for clearer tests if we refactored to make the estimate writers mockable dependenices of the repo and tested them separately...